From 60fc0fd9b08e3a5d9889cd43b6fe39593f0c54d9 Mon Sep 17 00:00:00 2001
From: Peter Simons <simons@cryp.to>
Date: Wed, 12 Apr 2023 22:31:08 +0200
Subject: [PATCH] postgresql-simple.cabal: drop dependency on obsolete
 bytestring-builder

We no longer support versions of 'bytestring' prior to version 0.10.4, because
these libraries lack the modules that provide bytestring-builder.
---
 postgresql-simple.cabal | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/postgresql-simple.cabal b/postgresql-simple.cabal
index 99b7b59..4677e54 100644
--- a/postgresql-simple.cabal
+++ b/postgresql-simple.cabal
@@ -82,7 +82,7 @@ library
   -- GHC bundled libs
   build-depends:
       base              >=4.6.0.0  && <4.19
-    , bytestring        >=0.10.0.0 && <0.12
+    , bytestring        >=0.10.4   && <0.12
     , containers        >=0.5.0.0  && <0.7
     , template-haskell  >=2.8.0.0  && <2.21
     , text              >=1.2.3.0  && <1.3 || >=2.0 && <2.1
@@ -93,7 +93,6 @@ library
   build-depends:
       aeson               >=1.4.1.0    && <1.6 || >=2.0.0.0 && <2.3
     , attoparsec          >=0.13.2.2   && <0.15
-    , bytestring-builder  >=0.10.8.1.0 && <0.11
     , case-insensitive    >=1.2.0.11   && <1.3
     , hashable            >=1.2.7.0    && <1.5
     , Only                >=0.1        && <0.1.1
